plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/XProjectManager.java

```diff
@@ -113,23 +125,30 @@ public void initialize(ProjectDescription description, IProjectConfig projectCon
 	/** Initial build reads the project state and resolves changes. */
 	public XBuildResult doInitialBuild(CancelIndicator cancelIndicator) {
 		Set<URI> changedSources = projectStateHolder.readProjectState(projectConfig);
-
+		// TODO distinguish between changed sources and deleted sources
```

Comment:
Could you add a ticket number?

Reply:
#1583 - I won't add it to the code, since I'll work on it now